Flutter/pdf-viewer/text-markup.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Text Markup Annotations in Flutter PDF Viewer widget | Syncfusion
+description: Learn here all about Text Markup Annotations in the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: Flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Text Markup Annotations in Flutter PDF Viewer widget (Syncfusion)
+
+The text markup annotations feature of `SfPdfViewer` allows you to add, remove and modify text markups in the PDF document. This is useful for making corrections or emphasizing important texts in the document. This section will go through the various types and functions available in PDF Viewer for working with text markup annotations.
+
+## Types of text markups
+
+The following text markup annotation types are now available in the PDF Viewer.
+1.	Highlight.
+2.	Squiggly.
+3.	Strikethrough.
+4.	Underline.
+
+## Add text markups
+
+This section will go through how to add text markup annotations to a PDF document interactively as well as programmatically to a PDF document.
+
+### Add text markups from annotation mode
+
+You can add text markup annotations to a PDF document by touch (or mouse down) and drag using `annotationMode` property of the PdfViewerController. The following steps explain how to add text markup annotation on a text in a PDF.
+
+1.	Set the AnnotationMode property of the SfPdfViewer to any text markups (say Highlight). It activates the highlight annotation mode on the control.
+2.	Place your finger (or mouse) on the text in the PDF document, where you want to start adding the text markup.
```

Comment:
Place your finger (or mouse) on the text in the PDF document where you want to start adding the text markup.